_episodes/03-use-registry.md

```diff
@@ -197,17 +214,9 @@ Zenodo is a general-purpose open access repository.
 > > If you go to the search box and look for "protein visualisation", you will see an entry like:
 > > TODO : add image
 > >
-> > That's it! You have published your tool in the development version or bio.tools. You are ready to go live and published yout tool for real! Remember bio.tools focuses on life sciences.
+> > That's it! You have published your tool in the development version or bio.tools. You are ready to go live and published your tool for real! Remember bio.tools focuses on life sciences.
 > {: .solution}
 {: .challenge}
 
 ## Wrap up
-We are increasing visibility, because we are supporting findability by adding the correct/good metadata -> Connect this with FAIR principles.
-
-
-**Instructor Notes / Setup**
-- [Local Installation of Zenodo](https://github.com/zenodo/zenodo/blob/master/INSTALL.rst)
-It may be interesting to have a local installation of zenodo to play around. The instructions using Docker are available on the link above.
-
-- [Bio-Linux](http://environmentalomics.org/bio-linux-software-list/)
-It is a final OS containing tools that have been already published, connected metadata, etc
+By adding good enough metadata to our research software, we are directly supporting its findability, thus increasing the overall visibility of the software. This is tied to the **findable** aspect of the FAIR principles mentioned in the introductory episode of this lesson.
```

Comment:
Metadate can also support accessibility if you include a license there, or interoperability if you include input/output data types or format. There might be some metadata supporting as well reusability.

That would be a nice exercise, asking attendees to map the metadata they have identified to FAIR principles... but maybe not enough time for it though...

Reply:
Fixed in the new commit. And excellent idea for an optional challenge @ljgarcia ! :)